Makefile

```diff
@@ -116,7 +116,7 @@ checks: format mypy lint tests
 
 docs: setup_ci $(install_tag) $(doc_files) setup.cfg
 	sphinx-apidoc -f -o sphinx/source/api src
-	make --directory=sphinx --file=Makefile html
+	make --directory=sphinx --file=Makefile clean html
```

Comment:
required to overwrite html sources every time (otherwise only new HTML sources will be created and changes in present ones won't be included)